[docs] Update Travis docs to use headless browsers instead of xvfb (#5331)
DevExpress/testcafe: docs/articles/documentation/guides/continuous-integration/travis.md

3. Add a `.travis.yml` configuration file to the root of your project. This file contains parameters and commands that instruct Travis CI how to execute your builds. For more information, see [Customizing the Build](https://docs.travis-ci.com/user/customizing-the-build).

>      For Node.js projects that require Firefox and Chrome installed, the `.travis.yml` file can have the following content:

Author: > What do you mean "can have the following content"? Why 'can'?

Should it rather be 'might have the following content' instead?  I moved this sentence from elsewhere and left it intact

     ```yaml
     language: node_js
     node_js: "stable"
  
     dist: bionic

     before_install:
        - stty cols 80

     addons:
        firefox: latest
        chrome: stable
     ```

     Commit and push this file to your repository.

## Step 3 - Add the `test` script to package.json

To test a project, Travis runs test scripts. For Node.js projects, the default test script is `npm test`.

> To tell npm how to run your tests, add the `test` script to the project's package.json file. Use the `:headless` postfix to run browsers headlessly.

Reviewer: ```suggestion
To tell npm how to run your tests, add the `test` script to the project's package.json file. Use the `:headless` postfix to run browsers in headless mode.
```

> > Important! Travis CI uses Ubuntu Server virtual machines that do not have regular graphical environment like Unity, GNOME or KDE installed. Modify the `testcafe` command to run browsers headlessly.

Reviewer 2: > Modify the `testcafe` command to run browsers headlessly.

Can we just write "Use the `:headless` prefix to run browsers headlessly."?

Because 'modify' would assume that a user already has a command without `:headless` and now they need to modify this command.

Author: Agreed. Although wouldn't this be a postfix rather than a prefix?

Reviewer 2: Oh, sorry, postfix of course 😄

Author: Addressed in [f15769d](https://github.com/DevExpress/testcafe/commit/f15769d4ea779dc1d133eee5f75d6114b3049e71)

> > Important! Travis CI uses Ubuntu Server virtual machines that do not have regular graphical environment like Unity, GNOME or KDE installed. Use the `:headless` postfix to run browsers headlessly.

Reviewer: ```suggestion
> Important! Travis CI uses Ubuntu Server virtual machines that do not have a graphic environment like Unity, GNOME or KDE. Use the `:headless` postfix to run browsers in headless mode.
```
